.github/workflows/alphabetise.yml

```diff
@@ -0,0 +1,51 @@
+name: Alphabetise extension list
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - antora-playbook.yml
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install packages
+        run: npm install -g yaml-sort
+
+      - name: Alphabetise
+        id: alpha
+        uses: mikefarah/yq@master
+        with:
+          path: antora-playbook.yml
+          cmd: '.content.sources |= sort_by(.url)'
+
+      - name: Overwite file
+        run: |
+          echo ${{ steps.alpha.outputs.result }} > antora-playbook.yml
+
+      - name: Create Pull Request
+        run: |
+          git checkout -b alpha-${{ github.run_id }}
+          git add antora-playbook.yml
+          git commit -m "Automatic alphabetisation"
+          git push
+          gh pr create --title "Automatic alphabetisation"
```

Comment:
I think you need the `-f` too, to fill the body too (or `-b`). You also need to set the GITHUB_TOKEN env

Reply:
See https://github.com/quarkiverse-release/quarkus-slack/blob/main/.github/workflows/perform-release.yml#L72